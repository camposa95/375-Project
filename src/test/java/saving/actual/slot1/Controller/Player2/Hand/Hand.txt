LUMBER: 0
ORE: 1
WOOL: 0
GRAIN: 2
BRICK: 0
