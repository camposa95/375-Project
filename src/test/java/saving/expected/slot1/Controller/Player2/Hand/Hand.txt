BRICK: 0
ORE: 1
WOOL: 0
LUMBER: 0
GRAIN: 2
